tools/serve/serve.py

```diff
@@ -264,7 +264,9 @@ def __call__(self, request, response):
             b"".join(response.iter_content(read_file=True)),
             b"<script>\n" +
             self.inject + b"\n" +
-            (b"// Remove the injected script tag from the DOM.\n"
+            (b"// Signal the presence of an injected script.\n"
+            b"window.INJECTED_SCRIPT=true;\n"
```

Comment:
@mfreed7 added a new global variable to signal the presence of  `--inject-script`. We check for it in the test function